docs/standards.md

```diff
@@ -25,7 +25,7 @@ XDM incorporates or builds a number of existing standards and specifications.
 Prior to XDM, Adobe defined and standardized the Extensible Metadata Platform (XMP) throughout its Creative Cloud products.
```

Comment:
Not just CC - but DC and others.  That is also why we took XMP to ISO for standardization (ISO 16684-1)